src/Nest/CommonOptions/Sorting/SortFormatter.cs

```diff
@@ -103,7 +103,7 @@ public void Serialize(ref JsonWriter writer, ISort value, IJsonFormatterResolver
 					for (var i = buffer.Offset; i < buffer.Count - 1; i++)
 						writer.WriteRawUnsafe(buffer.Array[i]);
 
-					if (buffer.Count > 0)
+					if (buffer.Count - buffer.Offset > 1)
```

Comment:
```suggestion
					// does the IGeoDistanceSort have other properties set i.e. is it more than simply {} ?
					if (buffer.Count > 2)
```

Reply:
`if (buffer.Count - buffer.Offset > 1)` will always be true because `buffer.Offset` is always 0 and `buffer.Count` will always be _at least_ 2.

Reply:
If `buffer.Offset` is always 0, why is it then that the for-loop uses `var i = buffer.Offset`?

Reply:
`i` could have been initialized to `0` instead, but I think using `buffer.Offset` is perhaps more clear about where the loop will start from?